src/android/com/eclipsesource/tabris/maps/MarkerPropertyHandler.java

```diff
@@ -36,6 +53,32 @@ private void setTitle(T marker, Properties properties) {
     }
   }
 
+  private void setImage(final T marker, Properties properties) {
+    if (properties.hasProperty(PROP_IMAGE)) {
+      List<Object> image = properties.getList(PROP_IMAGE, Object.class);
+      if (image == null) {
+        marker.setIcon(null);
+        return;
+      }
+      this.tabrisContext.getWidgetToolkit().getImageProvider().load(image, new ImageProviderListenerAdapter() {
+          @Override
+          public void onImageLoaded(Drawable drawable) {
+            marker.setIcon(getMarkerIconFromDrawable(drawable));
+            marker.updateMarker();
+          }
+      });
+    }
+  }
+
+  private BitmapDescriptor getMarkerIconFromDrawable(Drawable drawable) {
+    Canvas canvas = new Canvas();
```

Comment:
The Drawable is moste likely already a BitmapDrawable from which we could get the bitmap. The proposed code be used otherwise.